### PR TITLE
fix(style)

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

File: ethcore/res/ethereum/kotti.json [hunk 1]
```diff
@@ -1,855 +1,855 @@
...
-	"engine":{
-		"clique":{
-			"params":{
+	"name": "Kotti Testnet",
+	"dataDir": "kotti",
+	"engine": {
+		"clique": {
+			"params": {
 				"period":15,
 				"epoch":30000
```
Comment (soc1c) [inline ethcore/res/ethereum/kotti.json]: space here?

File: ethcore/res/ethereum/kotti.json [hunk 2]
```diff
@@ -1,855 +1,855 @@
...
-				"pricing":{
-					"linear":{
+		"0x0000000000000000000000000000000000000002": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "sha256",
+				"pricing": {
+					"linear": {
 						"base":60,
 						"word":12
```
Comment (soc1c) [inline ethcore/res/ethereum/kotti.json]: is there any rule where to have a space and where not?

File: ethcore/res/ethereum/kotti.json (hunk 1 above)
Comment (niklasad1) [reply ethcore/res/ethereum/kotti.json]: yes, I don't understand how this wasn't changed.

**Comment by niklasad1** [comment]
Should be fine now I had a bug in my regex

Verified by:
```$ git --no-pager grep -E "(\":[^ \t\r\n\v\f])+" | grep -E "(goerli|kotti|rinkeby)"```

**Pull request merged**